app/services/roles_service.rb

```diff
@@ -77,9 +78,11 @@ def create_default_roles!
 
     def create_default_hyrax_groups_with_roles!
       # Prevent Hyrax::Groups from being created in the public schema
-      return '`AccountElevator.switch!` into an Account before creating default Hyrax::Groups' if Site.instance.is_a?(NilSite)
+      return '`AccountElevator.switch!` into an Account before creating default Hyrax::Groups' if Site.instance
+                                                                                                      .is_a?(NilSite)
```

Comment:
One-line conditionals are nice, but sometimes it makes sense to break it into an `if...end` if the line is too long: 

```suggestion
      if Site.instance.is_a?(NilSite)
        return '`AccountElevator.switch!` into an Account before creating default Hyrax::Groups'
      end
```